docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -93,6 +95,39 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
      (product, warehouse, location), (warehouse), (product), (warehouse, product), ())`.
     The N elements of a `CUBE` specification results in 2^N `GROUPING SETS`.
 
+* **Partial Grouping Analytics**
+
+    Partial grouping analytics means there are both `group_expression` and `CUBE|ROLLUP|GROUPING SETS`
+    in GROUP BY clause. For example:
+    `GROUP BY warehouse, CUBE(product, location)` is equivalent to
```

Comment:
the example can focus on GROUPING SETS. We can just say that ROLLUP/CUBE is syntax sugar for GROUPING SETS and will be expanded to GROUPING SETS, then apply the Partial Grouping Analytics semantic.

Reply:
> the example can focus on GROUPING SETS. We can just say that ROLLUP/CUBE is syntax sugar for GROUPING SETS and will be expanded to GROUPING SETS, then apply the Partial Grouping Analytics semantic.

Done